projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/abstract-form-field.component.ts

```diff
@@ -21,6 +21,9 @@ export abstract class AbstractFormFieldComponent {
   @Input()
   public idPrefix = '';
 
+  @Input()
```

Comment:
@MateiHMCTS - is this input being used anywhere?

Reply:
As part of your question I had a more in-depth look.
With the current configuration on AAT it seems both (QM read and write view) are rendered with the read view (i.e. from here) -- even though the QM write view extends the AbstractFieldWriteComponent.

So I have access to caseReferences as injected here https://github.com/hmcts/ccd-case-ui-toolkit/blob/9c0fff46a0b44055dbd8cf1552d9fa25a9fb5863/projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.ts#L61

So question is : In the future, will the QM write view actually have write permissions?

The QM write extends the AbstractFieldWriteComponent, so typescript will complain when trying to access caseReference

Reply:
It's a good point - so, it will depend on whether Services will ever be allowed to configure the QM component with write permissions. Let's check with @ronaldmansveld